File: 0001 Code Chef/0019 Water Consumption/implementation/0005 kotlin/WaterConsumption.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput != null){
                    var water:Int? = stringInput.toIntOrNull();
                    if(water != null){
                        if(water >= 2000){
                            println("YES");
                        }else{
                            println("NO");
                        }
                    }
                }               
                count = count - 1;
            }
        }
    }
}

File: 0001 Code Chef/0019 Water Consumption/implementation/0010 nodejs/water_consumption.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 0;

    rl.on('line',(input)=>{
        let water = parseInt(input);
        if(water >= 2000){
            console.log("YES");
        }else{
            console.log("NO");
        }

        count = count + 1;
        if(count == test){
            rl.close();
        }
    })
})